lib/factory.js

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
```

Comment:
This stuff was copied over from work done last year, so we should include that in the copyright.

```suggestion
 * Copyright (c) 2022-2023 Digital Bazaar, Inc. All rights reserved.
```